Entity-ngsiv2-datamodels-schema.json

```diff
@@ -0,0 +1,238 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Entity-ngsiv2-datamodels-schema.json",
+  "title": "NGSIv2 Entity. Normalized Format for FIWARE Data Models",
+  "description": "NGSIv2 Entity. Normalized Format for FIWARE Data Models",
```

Comment:
The text is ok for title, but description should be more descriptive. Proposal:

"JSON Schema for NGSIv2 Entity with two restrictions. First, it only covers normalized format (keyValues and Values are not covered). Second, it only covers geo:json for entity location attributes (geo:point, geo:line, geo:box and geo:polygon are not covered). These restrictions are aligned with the usage of this JSON Schema for FIWARE Data Models."